src/java.base/share/classes/java/time/LocalDateTime.java

```diff
@@ -1807,7 +1807,10 @@ public ZonedDateTime atZone(ZoneId zone) {
      * chronology is also considered, see {@link ChronoLocalDateTime#compareTo}.
      *
      * @param other  the other date-time to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value, that is less than zero if this is before {@code other},
```

Comment:
This text is incorrect. It does not match the description above re chronologies.